test/integration/integration_test.py

```diff
@@ -0,0 +1,52 @@
+import subprocess, re
+from threading import Timer
+
+command = "velocitas exec runtime-local"
+regex_dapr_app = r".*You\'re up and running! Both Dapr and your app logs will appear here\.\n"
+regex_vdb = r".*Listening on \d+\.\d+\.\d+\.\d+:\d+"
+regex_mqtt = r".*mosquitto version \d+\.\d+\.\d+ running\n"
+regex_fc = r".*dbcfeeder: Connected to data broker"
+regex_vs= r".*DataBrokerFeederImpl: connected to data broker\."
+timeout_sec = 180
+def exec_until_matching_log(command: str, regex_dapr: re.Pattern, regex_specific: re.Pattern) -> bool:
+    successfully_running_dapr = False
+    successfully_running = False
+    if regex_dapr is not None:
+        built_dapr_regex = re.compile(regex_dapr)
+    else:
+        successfully_running_dapr = True
+    if regex_specific is not None:
+        built_specific_regex = re.compile(regex_specific)
+    else:
+        successfully_running = True
+    proc = subprocess.Popen(command.split(" "), stdout=subprocess.PIPE, bufsize=1, universal_newlines=True)
```

Comment:
use .run instead of .Popen to also make use of the timeout option of ".run" so you can remove your implemented timeout but have to add correct error handling for the ".run" (try except)